docs/quickstart/quickstart.ipynb

```diff
@@ -133,7 +133,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.7.10"
+   "version": "3.8.8"
```

Comment:
You need to update your environment, the Python version is now pinned to 3.7.10 so this change is incorrect

Reply:
Sure 😄

Reply:
Resolved 😄